simd/keccakf1600/example_test.go

```diff
@@ -1,79 +1,66 @@
 package keccakf1600_test
 
 import (
-	"github.com/cloudflare/circl/simd/keccakf1600"
-
 	"encoding/binary"
 	"fmt"
 
-	// Vendored copy of golang.org/x/crypto/sha3.
-	sha3 "github.com/cloudflare/circl/internal/shake"
+	"github.com/cloudflare/circl/simd/keccakf1600"
 )
 
-func Example() {
```

Comment:
Example is a magic name that will appear in godoc.  I don't think `Example_keccakf1600` will.